spec/services/mentor_at_school_periods/change_lead_provider_spec.rb

```diff
@@ -97,13 +97,11 @@
         end
 
         it "assigns the correct schedule for the new training period" do
-          travel_to Date.new(2025, 9, 1) do
-            subject
+          subject
 
-            new_training_period = mentor_at_school_period.training_periods.ongoing.first
-            expect(new_training_period.schedule.identifier).to eq("ecf-standard-september")
-            expect(new_training_period.schedule.contract_period_year).to eq(2025)
-          end
+          new_training_period = mentor_at_school_period.training_periods.ongoing.first
+          expect(new_training_period.schedule.identifier).to eq("ecf-standard-september")
+          expect(new_training_period.schedule.contract_period_year).to eq(2025)
```

Comment:
Do we need to use the current year here rather than a hard coded one? Will this fail in 2026?